src/exo/API_scheduling.py

```diff
@@ -1277,6 +1451,10 @@ def rearrange_dim(proc, buf_cursor, permute_vector):
         buf_cursor      - cursor pointing to an Alloc statement
                           for an N-dimensional array
         permute_vector  - a permutation of the integers (0,1,...,N-1)
+        rc              - (bool) whether to return relevant cursors
+
+    relevant cursors:
+        alloc           - a cursor to the modifed alloc
 
```

Comment:
Similar to the above comment, this op should be renamed to `rearrange_dims` most likely